winforms/src/toga_winforms/window.py

```diff
@@ -210,7 +154,15 @@ def close(self):
         self._is_closing = True
         self.native.Close()
 
-    def winforms_resize(self, sender, args):
-        if self.interface.content:
-            # Re-layout the content
-            self.interface.content.refresh()
+    def resize_content(self):
+        vertical_shift = 0
+        if self.toolbar_native:
+            vertical_shift += self.toolbar_native.Height
+        if self.native.MainMenuStrip:
+            vertical_shift += self.native.MainMenuStrip.Height
+
+        self.native_content.Location = Point(0, vertical_shift)
+        super().resize_content(
```

Comment:
This is the essentially the reason I'm questioning the composition vs inheritance approach - composition gives a clear distinction between the process of resizing the window, and the fact this requires resizing the container.